spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/support/FilterConfig.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2013-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.springframework.cloud.gateway.support;
+
+/**
+ * Filter global const
+ */
+public class FilterConfig {
+
+    private FilterConfig(){}
+
+	public static final String DEFAULT_FILTERS = "defaultFilters";
```

Comment:
I dont think this class is necessary, lets keep the definition of `DEFAULT_FILTERS` in `RouteDefinitionRouteLocator`

Reply:
`RedisRateLimiter` need `DEFAULT_FILTERS ` too. So let it import `RouteDefinitionRouteLocator. DEFAULT_FILTERS` ?

Reply:
yes

Reply:
done